services/GoogleSitemapService.cfc

```diff
@@ -42,6 +42,7 @@ component {
 		googleSitemap.xmlRoot = xmlElemNew( googleSitemap, "urlset" );
 		googleSitemap.xmlRoot.XmlAttributes.xmlns = "http://www.sitemaps.org/schemas/sitemap/0.9"
 
+		arguments.logger.info( "Starting to rebuild XML sitemap for [#ArrayLen(arguments.pages)#] pages" );
```

Comment:
You should add a check to see if the logger is passed into the arguments. It is possible that this code could be called in a context outside of task manager. Something like:

```
var canLog = arguments.keyExists( "logger" );
var canInfo = canLog && arguments.logger.canInfo();

...

if ( canInfo ) { arguments.logger.info( ... ); }
```